src/locales/en.ts

```diff
@@ -422,6 +422,16 @@ export default {
     hotspot_label: 'Hotspot',
     last_activity: 'LAST REPORTED ACTIVITY: {{activity}}',
     label_error: 'You do not have enough HNT in your account.',
+    scan: {
+      title: 'Find a QR Code to Scan',
+      send: 'Send HNT',
+      send_description: 'Quickly scan a Helium address to send HNT.',
+      burn: 'Burn HNT to DC',
+      burn_description: 'HNT can be burned into Data Credits to pay for device network connectivity. DCs are non-transferable.',
+      view: 'View My QR Code',
+      view_description: 'Save your QR code for others to save and send payments.',
```

Comment:
I was planning on having this slide up a modal that shows your own QR code. like if you hit the QR code button thinking that it would show yours rather than scan one. I don't know if that changes your description here or not

Reply:
Hm, I accessed this UI after hitting Send > QR icon, so that's where the copy's context is coming from. Is there another entry point?

Reply:
the main entry point for your qr code is under the activity list when you press "receive". But figured this would be a good secondary entry point. Mainly I was just thinking rather than "Save" it could be like "Show your QR code to receive payments from others" or something? And rather than "Learn more" it could be "View QR Code" to launch a modal with it?